src/components/Cheatsheet.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { PropsWithChildren } from 'react'
+import * as rb from 'react-bootstrap'
+import { Link } from 'react-router-dom'
+import { Trans, useTranslation } from 'react-i18next'
+
+interface CheatsheetProps {
+  show: boolean
+  onHide(): () => void
+}
+
+type NumberedProps = {
+  number: number
+}
+
+function Numbered({ number }: { number: number }) {
+  return <div className="numbered">{number}</div>
+}
+
+function ListItem({ number, children }: PropsWithChildren<NumberedProps>) {
```

Comment:
I think you can take this one step further

```
interface ListItemProps {
  number: number
  transaltionKey: string
  descriptionKey: string
}

function ListItem({ number, children, transaltionKey, descriptionKey }: PropsWithChildren<ListItemProps>) {
  const { t } = useTranslation()
  return (
    <rb.Stack className="cheatsheet-list-item" direction="horizontal" gap={3}>
      <div className="numbered">{number}</div>
      <rb.Stack gap={0}>
        <h6>
          <Trans i18nKey={transaltionKey}>{children}</Trans>
        </h6>
        <div className="small text-secondary">{t(descriptionKey)}</div>
      </rb.Stack>
    </rb.Stack>
  )
}
```

```
<ListItem number={1} transaltionKey="cheatsheet.item_1.title" descriptionKey="cheatsheet.item_1.description">
    <Link to="/receive">Fund</Link> your wallet first.
</ListItem>

```

Reply:
Passing translation keys strikes me as unnecessarily restrictive.
It is so little code - feels like over-engineering to me for now. Do not get me wrong - I _love_ to over-engineer.. : D
But feel free to add any improvements once the PR is merged!